server/handlers/flightFetcher.js

```diff
@@ -236,7 +236,6 @@ class FlightFetcher {
       airport: arrivalAirport
     };
   }
-
```

Comment:
r u sure there isn't a better way? maybe we should schedule a meeting to discuss a different approach. call me k bye.